docs/12_0_0/core/contentsecuritypolicy.md

```diff
@@ -34,6 +34,20 @@ Browsers will report CSP violations to a user-defined HTTP endpoint using `repor
 </context-param>
 ```
 
+When using Primefaces in a Portlet context you might want to generate the Nonce and set the CSP header outside of 
+Primefaces. Using `policyProvided` will make the code CSP complient while not setting the CSP Header.
+
+```xml
+<context-param>
+    <param-name>primefaces.CSP</param-name>
+    <param-value>policyProvided</param-value>
+</context-param>
+<context-param>
+    <param-name>primefaces.CSP_REPORT_ONLY_POLICY</param-name>
+    <param-value>report-uri /my-csp-reports</param-value>
```

Comment:
Actually if you do need REPORT_ONLY_POLICY shouldn't we make this a real world example how you would use it with Portlets?

Reply:
The REPORT_ONLY_POLICY is not required, it was a copy-paste error.